wordpress-mobile/WordPress-iOS
# Fix Acknowledgements

Since migrating to SPM, we've lost the acknowledgements for our third-party libraries.

This PR fixes that.

It also:
- Cleans up some URL stuff around the app
- Presents URLs in the navigation stack for "About" subpages. Ideally it would do that for the root items too, but I'd need to modify AutomatticAbout for that, so we'll take the half-win for now

To test:

Check out the "Me > About > Acknowledgements" section of the app, and note that the sibling links on that screen also present in a navigation view now.

## Regression Notes
1. Potential unintended areas of impact
Other "About" screens could break

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tried each out by hand

3. What automated tests I added (or what prevented me from doing so)
Not worth adding UI snapshot tests

PR submission checklist:


Testing checklist:
